.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -63,7 +76,7 @@ If possible, use https://new.docusaurus.io to create a CodeSandbox reproducible
 - Public site url:
```

Comment:
```suggestion
- Public site URL:
```

While we're fixing things up, URL is an abbreviation, and in English we capitalize abbreviations. 😄